consumer_test.go

```diff
@@ -3,6 +3,7 @@ package consumer
 import (
 	"context"
 	"fmt"
+	"github.com/harlow/kinesis-consumer/store/memory"
```

Comment:
Can we call this “store” as the import. Then we’ll call “store.New()” instead of memory.new